.github/workflows/nightly_fuzzing.yml

```diff
@@ -40,7 +40,7 @@ jobs:
         id: tests
         continue-on-error: true # To send a notification if the tests fail
         run: |
-          uv run --reinstall pytest -n logical --durations=0 -v -s --log-cli-level=DEBUG --no-skip-cached-tests
+          uv run --reinstall pytest -n logical --durations=0 -v -s --log-cli-level=DEBUG --no-skip-cached-tests --ignore-glob=cairo/tests/ef_tests/*
```

Comment:
we might want a specific job in for ef-tests that runs every night.
Even if there is only state_transition for now, it should be implemented.
Also in this file a 64CPU should be used

Reply:
https://github.com/kkrt-labs/keth/issues/837